applications/blink_raw/targets/freertos.mips4k.pic32mx.lccbuffer/pic32mx_cf_init.c

```diff
@@ -0,0 +1 @@
+../../../../boards/mc-pic32mx-lccbuffer/pic32mx_cfg_init.c
```

Comment:
This is a typo, I needed to redo the symlinks in Windows after they were copied to a new directory.